src/index.ts

```diff
@@ -90,6 +118,14 @@ class ServerlessPlugin {
     // Remove temp build folder
     fs.removeSync(path.join(this.originalServicePath, buildFolder))
   }
+
+  async cleanup(): Promise<void> {
```

Comment:
Could we somehow avoid that redundancy as the same code is also part of the previous function?